app/assets/javascripts/articles.js

```diff
@@ -1,13 +1,21 @@
 App.articleQueue.onIncoming = function(data) {
-  var alertDiv = $('#floating-alert');
+  var alertDiv = document.querySelectorAll('#floating-alert');
 
   if (alertDiv.length === 0) {
-    alertDiv = $('<div id="floating-alert" class="alert alert-floating alert-notice">)');
-    $('body').prepend(alertDiv);
+    alertDiv = document.createElement('div');
+    alertDiv.id = 'floating-alert';
+    alertDiv.className = 'alert alert-floating alert-notice';
```

Comment:
i normally use `alertDiv.setAttribute()` and `alertDiv.classList.add()`, etc for this sort of stuff. In this context i don't think it makes a difference so... 🤷‍♂. just my two cents.

Reply:
Thanks! You're right, that's better style. I made a commit with the changes you suggested: https://github.com/wbholland/website/commit/40b5707597c4db1d3930a620715bda37718d9b0b